[docker] Add note about mounting "/etc/os-release" (#12749)
DataDog/documentation: content/en/agent/docker/_index.md

**Note**: If you're using a different registry besides GCR or ECR-public, make sure to update the image.

> **Note**: For some features provided by system-probe (network monitoring, security agent, oom_kill check, etc.) you will also need mount the `/etc/os-release` file with `-v /etc/os-release:/host/etc/os-release:ro`. If your Linux distribution does not include an `/etc/os-release` file, mount the equivalent one provided (`/etc/redhat-release`, `/etc/fedora-release`, etc.) instead.

Reviewer: ```suggestion
**Note**: For some features provided by system-probe, including network monitoring, security agent, and oom_kill check, you must also mount the `/etc/os-release` file with `-v /etc/os-release:/host/etc/os-release:ro`. If your Linux distribution does not include an `/etc/os-release` file, mount the equivalent one provided, for example `/etc/redhat-release` or `/etc/fedora-release`.
```
